### PR TITLE
Create INSTALL-XAMPP.md

### PR DESCRIPTION
**Pull request by zuphilip**
Previous version at https://gist.github.com/zuphilip/b8a1d393c39647cb73b1ed99039c33b4

### CODE REVIEW

**Comment by zuphilip** [comment]
Thanks to @vossviola for pointing at the typos. Let me know about any further remarks before merging.

File: INSTALL-XAMPP.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+It may be handy to install the tools on
+windows for testing and/or developing.
+For such purposes you can install and
+run the tools in cominbation with
+XAMPP.
+
+ 1. Install XAMPP, e.g. in the folder `C:\xampp\`
+ 2. Clone malibu in the `htdocs` subfolder
+ 3. Download jQuery and place it in `.\htdocs\malibu\isbn`
+ 4. Install [`yaz`](http://www.indexdata.com/yaz) for windows.
```
Comment (zuphilip) [inline INSTALL-XAMPP.md]: + and make sure that the `bin` directory of `YAZ` is part of the PATH environment variable.

File: INSTALL-XAMPP.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Some components are available for 64-bit and/or x86 (32-bit) architecture. It is important that you use the compatible versions for all components, e.g. only use x86-architecture versions.
+ 
+For me the following components are working together:
+ * XAMPP 3.2.2 with PHP 7.0.8 (only available on x86-architecture)
+ * yaz 5.15.2 for x86-architectures
+ * php_yaz.dll for x86-architectures
+
+## Open problems
+
+Starting XAMPP there is an error because `libiconv.dll` is not found...
```
Comment (zuphilip) [inline INSTALL-XAMPP.md]: No, that is actually unrelated here.

**Comment by vossviola** [comment]
we could think about adding two more URLs for "full service": 
1. https://www.apachefriends.org
3. http://jquery.com/download/, or e.g. http://data.bib.uni-mannheim.de/malibu/isbn/jquery-2.1.1.min.js

File: INSTALL-XAMPP.md (hunk 1 above)
Comment (vossviola) [inline INSTALL-XAMPP.md]: workaround may be: copy dll files (not the ic*40.dll files, these are not needed) from \YAZ\bin to .\apache\bin; YAZ may be deinstalled after that
(cf. https://blog.verweisungsform.de/2009-08-17/yaz-php-apache-und-windows/)

**Comment by vossviola** [comment]
perhaps add a step before 7. like "for using the ISBN tool copy conf.example.php to conf.php and enter data" [hatten wir das thema nicht hier schon irgendwo? :)]

File: INSTALL-XAMPP.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+For such purposes you can install and
+run the tools in cominbation with
+XAMPP.
+
+ 1. Install XAMPP, e.g. in the folder `C:\xampp\`
+ 2. Clone malibu in the `htdocs` subfolder
+ 3. Download jQuery and place it in `.\htdocs\malibu\isbn`
+ 4. Install [`yaz`](http://www.indexdata.com/yaz) for windows.
+ 5. Download [`php_yaz.dll`](http://ftp.indexdata.dk/pub/phpyaz/windows/) (and possibly the other dll's) from IndexData
+ 5. Copy `php_yaz.dll` (and possibly the other dll's) into `.\php\ext`
```
Comment (zuphilip) [inline INSTALL-XAMPP.md]: Das Zählen muss ich nochmals üben... :laughing:

Comment (vossviola) [inline INSTALL-XAMPP.md]: :)
